clang/include/clang/Sema/DeclSpec.h

```diff
@@ -2776,6 +2797,53 @@ struct FieldDeclarator {
         BitfieldSize(nullptr) {}
 };
 
+class ContractSpecifiers {
```

Comment:
TODO: Remove this, I actually haven't used it yet.

Though I haven't fixed the parsing yet.